docs/personalization/enable_personalization.md

```diff
@@ -161,33 +160,43 @@ Place the following code snippet in the `<head>` section of your header template
 ### Check whether the bundle provides REST data
 
 You can verify the import controller of the bundle by calling the local API.
-Use the `Accept` header; you may need to add an `Authorization` header if authentication is required.
+As the API uses token based authorization you first need a valid bearer token.
+
+When you publish a content item a bearer token is created and saved to the `ibexa_token` db table.
+
+Additionally a POST request is send to the Personalization Engine,  containing the token
+and the Rest URL where the Personalization Engine can fetch the changed Content.
+
+The `BEARER_TOKEN` is the newest one in `ibexa_token` table having `type=1` and `identifier=update`. The token has a default lifetime of one day.
+
+You can use this token to check what is provided to the Personalization Engine:
 
-To check whether the `content` endpoint is working as expected, perform the following request:
 
 ```
-GET http://<yourdomain>/api/ezp/v2/ibexa_personalization/v1/content/{contentId}
-Accept application/vnd.ez.api.Content+json
-Authorization Basic xxxxxxxx
+curl --location '{PERSONALIZATION_HOST_URI}/api/ibexa/v2/personalization/v1/content/id/{contentId}?lang={comma_separated_languages}' \
```

Comment:
Does it need a domain dedicated to personalization? If yes, `PERSONALIZATION_HOST_URI` need a definition; if not, it need to me more generic. Why not `HOST_URI` or `YOUR_HOST_URI`?

Reply:
@adriendupuis this is a variable mentioned during personalisation configuring process https://doc.ibexa.co/en/latest/personalization/enable_personalization/